run/action.yaml

```diff
@@ -105,18 +92,23 @@ runs:
 
     - name: Install Nix
       uses: cachix/install-nix-action@v18
+      env:
+        GITHUB_PAT: ${{ inputs.github_pat }}
+        NIX_KEY_PATH: ${{inputs.key_path}}
       with:
         extra_nix_config: |
           accept-flake-config = true
           trusted-users = runner
-          access-tokens = github.com=${{ inputs.github_pat }}
-          secret-key-files = ${{ env.NIX_KEY_PATH }}
+          access-tokens = github.com="$GITHUB_PAT"
```

Comment:
this isn't gonna work, its not a shell snippet

Reply:
Yeah, just realized this a little later.  It's fixed now. Thx! :smile: